src/nav/navReducer.js

```diff
@@ -11,17 +11,7 @@ import { INITIAL_FETCH_COMPLETE, ACCOUNT_SWITCH, LOGIN_SUCCESS, LOGOUT } from '.
  * Private; exported only for tests.
  */
 export const getStateForRoute = (route: string): NavigationState => {
-  // TODO: this is kind of a hack!  Refactor to a better way.
```

Comment:
Aha, interesting—I think this is probably a case of there being a previous revision where something different came after "a", such that "a" was definitely correct, but then I put that `initial: true` bit after the "a" and forgot to change "a" to "an". Possibly because the backtick made the incorrectness not stand out to me as much? 🤔